.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+### :bust_in_silhouette: Who
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/alopezna5/mASAPP_CI/pullRequest?prid=5264864)